cpp/src/io/csv/reader_impl.cu

```diff
@@ -106,13 +107,13 @@ class selected_rows_offsets {
 /**
  * @brief Removes the first and Last quote in the string
  */
-string removeQuotes(string str, char quotechar)
+std::string_view remove_quotes(std::string_view str, char quotechar)
 {
   // Exclude first and last quotation char
   size_t const first_quote = str.find(quotechar);
-  if (first_quote != string::npos) { str.erase(first_quote, 1); }
+  if (first_quote != string::npos) { str = str.substr(first_quote + 1); }
   size_t const last_quote = str.rfind(quotechar);
-  if (last_quote != string::npos) { str.erase(last_quote, 1); }
+  if (last_quote != string::npos) { str.substr(0, last_quote); }
```

Comment:
```suggestion
  if (last_quote != string::npos) { str = str.substr(0, last_quote); }
```
Should probably just incorporate the code from my comment above.
https://github.com/rapidsai/cudf/pull/17734/files#r1918642398

Reply:
thanks!